STARTER/.gitignore

```diff
@@ -0,0 +1,53 @@
+# Node modules #
```

Comment:
This file contains a lot of stuff. I think it should be project specific, not user machine specific.